.github/workflows/dispatch-matrix-test-on-comment.yml

```diff
@@ -17,11 +17,20 @@ jobs:
         with:
           minimum-permission: "write"
 
+      - name: Generate token
+        id: generate-token
+        uses: actions/create-github-app-token@eaddb9eb7e4226c68cf4b39f167c83e5bd132b3e
```

Comment:
`actions/create-github-app-token` an official action maintained by GitHub, so we could just pin this to a tag.

Reply:
I copied what is used elsewhere in the repo. Checked in with Remco as to why, and it was because of confusion over whether GitHub maintained the action or not. As it is clear we do, I've switched to using `v1` as the latest major tag.